1001-2000/LIT1621Homily.xml

```diff
@@ -1,50 +1,83 @@
-type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1621Homily">
-   <teiHeader xml:base="https://betamasaheft.eu/">
-      <fileDesc>
-         <titleStmt>
-            <title xml:id="t1">Homily on Abraham for Good Friday</title>
-            <author>Jacob of Serug</author>
-            <editor role="generalEditor" key="AB"/>
-            <funder>Akademie der Wissenschaften in Hamburg</funder>
-         </titleStmt>
-         <publicationStmt>
-            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
-            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
-            <pubPlace>Hamburg</pubPlace>
-            <availability>
-               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
-                  <p>
-                This file is licensed under the Creative Commons
-                Attribution-ShareAlike 4.0. </p>
-               </licence>
-            </availability>
-         </publicationStmt>
-         <sourceDesc>
-            <p>Work of the literatures of Ethiopia and Eritrea</p>
-            <p>A TEI file converted from the ETHIO-authority google spreadsheet</p>
-         </sourceDesc>
-      </fileDesc>
-      <encodingDesc>
-         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
-            <xi:fallback>
-               <p>Definitions of prefixes used.</p>
-            </xi:fallback>
-         </xi:include>
-      </encodingDesc>
-      <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
-      </profileDesc>
-      <revisionDesc>
-         <change who="PL" when="2016-03-21"> Created file from google
-            spreadsheet </change>
-         <change who="ES" when="2016-02-09">CREATED: text record</change>
-      </revisionDesc>
-   </teiHeader>
-   <text xml:base="https://betamasaheft.eu/">
-      <body>
-         <ab/>
-      </body>
-   </text>
-</TEI>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1621Homily">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+    <fileDesc>
+      <titleStmt>
+        <title xml:id="t1">ድርሳን፡ ዘደረሰ፡ ቅዱስ፡ ወብፁዕ፡ አባ፡ ያዕቆብ፡ ኤጲስ፡ ቆጶስ፡ ዘስሩግ፡ በእንተ፡ አብርሃም፡ አቡነ፡ ዐርከ፡ እግዚአብሔር፡ ወበእንተ፡ አቀርቦቱ፡ ለይስሐቅ፡ ወልዱ፡ ወእፎ፡ ቤዘዎ፡ እግዚአብሔር፡ በበግዕ፡</title>
```

Comment:
አቅርቦቱ?

Reply:
Probably... But I will send you the original via Skype